articles/azure-functions/functions-create-github-webhook-triggered-function.md

```diff
@@ -37,7 +37,7 @@ Learn how to create a function that is triggered by an HTTP webhook request with
 
 ![Function app successfully created.](./media/functions-create-first-azure-function/function-app-create-success.png)
 
-Next, you create a function in the new function app.
+Next, you will create a function in the new function app.
```

Comment:
While I agree with you, our Azure-wide authoring guidance is to NOT use a future tense and this change will lower the topic’s score 😀

Reply:
Thank you for pointing out that authoring guidance. I've reverted.